src/components/pages/FamilyMembers/Family.jsx

```diff
@@ -1,15 +1,13 @@
 import React, { useEffect, useState } from 'react';
 import { axiosWithAuth } from '../../../api/axiosWithAuth';
 import LoadingComponent from '../../common/LoadingComponent';
-import { useParams } from 'react-router-dom';
 import MaterialTable, { MTableToolbar } from 'material-table';
 import { useHistory } from 'react-router-dom';
 import { tableIcons } from '../../../utils/tableIcons';
 import { Button } from '@material-ui/core';
 
-const FamilyMembers = () => {
+const FamilyMembers = ({ familyId }) => {
   const history = useHistory();
-  const params = useParams();
   const [loading, setLoading] = useState(true);
   // State below is unused
   //const [familyInfo, setFamilyInfo] = useState({});
```

Comment:
Could remove this, only flaw I see

Reply:
For sure - I was saving it in case we needed it but It makes sense to remove it now. Thanks!